### PR TITLE
Remove space ids

### PR DESCRIPTION
**Pull request by sdrave**

### CODE REVIEW

**Review by HenKlei — APPROVED** [review]

**Review by pmli — CHANGES_REQUESTED** [review]
Files `reduced_model.out` and `results.out` have been added. I guess this was not on purpose. I had similar issues with those files, perhaps they could be added to `.gitignore`.

**Comment by sdrave** [comment]
> Files `reduced_model.out` and `results.out` have been added. I guess this was not on purpose. I had similar issues with those files, perhaps they could be added to `.gitignore`.

Ouch. They were actually listed in `.gitignore`, but only for specific paths. I have changed that.

File: .gitignore [hunk 1]
```diff
@@ -9,10 +9,10 @@ src/pymor/discretizers/builtin/inplace.c
...
 src/pymor/discretizers/builtin/grids/_unstructured.c
 src/pymordemos/minimal_cpp_demo/bindings_generated.cpp
-src/pymordemos/reduced_model.out
-src/pymordemos/results.out
-src/pymortests/reduced_model.out
-src/pymortests/results.out
+reduced_model.out
+results.out
+reduced_model.out
+results.out
```
Comment (pmli) [inline .gitignore:15]: Two lines could be removed now.

**Review by pmli — CHANGES_REQUESTED** [review]
Looks good to me, except that `DuneXTMatrixOperator` documents `source_id` and `range_id`.

File: .gitignore (hunk 1 above)
Comment (sdrave) [reply .gitignore]: Done.

**Review by pmli — APPROVED** [review]

**Pull request merged**